plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -105,7 +105,9 @@
 import io.trino.spi.type.Type;
```

Comment:
The commit message should mention what's fixed since there are changes in non-test code.